<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Addnew</class>
 <widget class="QDialog" name="Addnew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../build-MY_store-Desktop-Debug/new_forgot_password.JPG</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_username">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Enter new username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Enter new Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_nickname">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_friendname">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_unclename">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>390</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
